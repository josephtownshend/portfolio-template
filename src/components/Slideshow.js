import React from 'react';

export default class Slideshow extends React.Component {

  render() {
    return(
      <div> SlideShow!</div>
    )
  }
}
