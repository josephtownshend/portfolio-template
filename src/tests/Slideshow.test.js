import React from 'react';
import { configure } from 'enzyme';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Slideshow from '../components/Slideshow';

configure({ adapter: new Adapter() });

it('shallow renders without crashing', () => {
  shallow(<Slideshow />)
});

it('renders correctly', () => {
  const tree = renderer.create(<Slideshow />).toJSON();
  expect(tree).toMatchSnapshot();
})
