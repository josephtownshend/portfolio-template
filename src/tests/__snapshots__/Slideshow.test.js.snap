// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div>
    <img
      alt="Lauren Keeley"
      src="http://www.fruttagallery.com/wp-content/uploads/1.LaurenKeeley_InaYear_2015_Frutta.jpg"
      style={
        Object {
          "width": 900,
        }
      }
    />
  </div>,
  <div>
    <button
      className="Button"
      onClick={[Function]}
    >
      Previous
    </button>
    <button
      className="Button"
      onClick={[Function]}
    >
      Next
    </button>
  </div>,
]
`;
